src/test/java/org/folio/eusage/reports/MainVerticleTest.java

```diff
@@ -1238,6 +1285,40 @@ public void testPostTenantOK(TestContext context) {
     titlesAr = resObject.getJsonArray("titles");
     context.assertEquals(9, titlesAr.size());
 
+    response = RestAssured.given()
+        .header(XOkapiHeaders.TENANT, tenant)
+        .header(XOkapiHeaders.URL, "http://localhost:" + MOCK_PORT)
+        .get("/eusage-reports/report-titles?providerId=" + usageProviderId + "&query=kbTitleId=" + goodKbTitleId)
+        .then().statusCode(200)
+        .header("Content-Type", is("application/json"))
+        .extract();
+    resObject = new JsonObject(response.body().asString());
+    titlesAr = resObject.getJsonArray("titles");
+    context.assertEquals(1, titlesAr.size());
+    context.assertEquals(goodKbTitleId.toString(), titlesAr.getJsonObject(0).getString("kbTitleId"));
+
+    response = RestAssured.given()
+        .header(XOkapiHeaders.TENANT, tenant)
+        .header(XOkapiHeaders.URL, "http://localhost:" + MOCK_PORT)
```

Comment:
This repeated code
```
RestAssured.given()
        .header(XOkapiHeaders.TENANT, tenant)
        .header(XOkapiHeaders.URL, "http://localhost:" + MOCK_PORT)
```
should be

* extracted into a helper method,
* extracted into a requestSpec https://github.com/rest-assured/rest-assured/wiki/Usage#specification-re-use
* or set as a default request spec https://github.com/rest-assured/rest-assured/wiki/Usage#default-values